v0/destinations/customerio/transform.js

```diff
@@ -144,6 +145,14 @@ function responseBuilder(message, evType, evName, destination, messageType) {
     }
     endpoint = IDENTITY_ENDPOINT.replace(":id", userId);
     requestConfig = defaultPutRequestConfig;
+  } else if (evType === EventType.ALIAS) {
+    // ref : https://customer.io/docs/api/#operation/merge
+    endpoint = MERGE_USER_ENDPOINT;
+    requestConfig = defaultPostRequestConfig;
+    rawPayload.primary = {};
+    rawPayload.primary.id = userId;
+    rawPayload.secondary = {};
+    rawPayload.secondary.id = message.previousId;
```

Comment:
Can we throw error if either of `userId` and `previousId` is not valid as per the docs both are required fields?

Reply:
done